composer.json

```diff
@@ -36,7 +36,10 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Tests\\": "tests/"
+            "Tests\\": "tests/",
+            "Tests\\Expected\\Records\\": "tests/fixtures/expected",
+            "Tests\\Expected\\": "tests/fixtures/expected",
+            "Tests\\Expected\\Records\\Nested\\": "tests/fixtures/expected"
```

Comment:
For newer composer versions, you apparently have to call these paths out specifically for the tests to be able to find these classes.